admin/templates/pre-reg/prereg.html

```diff
@@ -0,0 +1,70 @@
+        </tr>
+    </thead>
+    <tbody class="scripted" data-bind="foreach: sortedDrafts">
+        <tr data-bind="attr: {id: $index}, event: {mouseover: highlightRow, mouseout: unhighlightRow, click: goToDraft}">
+          <td data-bind="text: title" class="text-center" id="submission_title"></td>
+          <td data-bind="text: fullname" class="text-center" id="name"></td>
+          <td data-bind="text: username" class="text-center" id="email"></td>
+          <td data-bind="text: initiated" class="text-center" id="begun"></td>
+          <td data-bind="text: updated" class="text-center" id="submitted"></td>
+          <!-- Proof of Publication -->
+          <td class="text-center" id="proof_of_pub">
+            <select data-bind="options: proofOfPub.proofOfPubList, value:proofOfPub.proofOfPub, click: proofOfPub.editItem, clickBubble: false"></select>
+          </td>
+          <!-- Payment Sent -->
+          <td class="text-center" id="payment_sent">
+            <div data-bind="visible: !paymentSent.edit()">
+              <select data-bind="options: paymentSent.paymentSentList, value:paymentSent.paymentSent, click: paymentSent.editItem, clickBubble: false"></select>
+            </div>
+            <div data-bind="valueUpdate: 'afterkeydown', enterkey: paymentSent.stopEditing, visible: paymentSent.edit()">
+              <div><input type="radio" value="no" data-bind="checked: paymentSent.paymentSent, attr:{name: 'paymentSent' + $index()}"/>No</div>
+              <div><input type="radio" value="yes" data-bind="checked: paymentSent.paymentSent, attr:{name: 'paymentSent' + $index()}"/>Yes</div>
+            </div>
+          </td>
+          <!-- Notes -->
+          <td class="text-center" id="notes">
+            <div data-bind="visible: !notes.edit()">
+              <span data-bind="text: notes.notes"></span><i data-bind="click: notes.editItem, clickBubble: false, event: {mouseover: notes.enlargeIcon, mouseout: notes.shrinkIcon}, attr: {class: 'fa fa-pencil'}" style="margin-left: 10px"></i>
+            </div>
+            <div data-bind="valueUpdate: 'afterkeydown', enterkey: notes.stopEditing, visible: notes.edit()">
+              <div><input type="text" data-bind="value: notes.notes"/></div>
+            </div>
+          </td>
+          <!-- Status -->
+          <td data-bind="text: status" class="text-center" id="status"></td>
+          <!-- Assignee -->
+          <td class="text-center" id="assignee">
+            <select data-bind="visible: adminPermission() ,value: assignee.assignee, options: assignee.reviewers, click: assignee.editItem, clickBubble: false"></select>
+            <span data-bind="visible: !adminPermission(), text: assignee.assignee, click: assignee.editItem, clickBubble: false"></span>
+          </td>
+        </tr>
+    </tbody>
+  </table>
+</div>
+
+{% render_bundle 'prereg-admin-page' %}
```

Comment:
There's a bottom_js block that this could be inside of. The benefit of sticking js inside of blocks instead of inside the main content block is you might want to turn off just the js one day without duplicating the entire body of the page.